src/Lighthouse.jl

```diff
@@ -18,6 +20,9 @@ export confusion_matrix, accuracy, binary_statistics, cohens_kappa, calibration_
 include("classifier.jl")
 export AbstractClassifier
 
+include("row.jl")
+# TODO: export EvaluationRow ?
```

Comment:
Should we export EvaluationRow?
```suggestion
export EvaluationRow
```

Reply:
Will do, although I'm not sure I understand the rationale?

Reply:
Ah, gotcha! Thanks for the explanation.